src/lib/select/select.ts

```diff
@@ -175,6 +177,27 @@ export const _MatSelectMixinBase:
         mixinDisableRipple(mixinTabIndex(mixinDisabled(mixinErrorState(MatSelectBase))));
 
 
+/** Possibe rendering strategies for the MatSelect panel. */
+export enum MatSelectRenderingStrategy {
```

Comment:
How about `MatSelectOptionsRenderOptions`? I would want o only use "Strategy" when we're using the actual strategy pattern.